src/writers/codecov-writer.cc

```diff
@@ -134,6 +135,12 @@ class CodecovWriter : public WriterBase
 				IReporter::LineExecutionCount cnt = m_reporter.getLineExecutionCount(file->m_name, n);
 				std::string hitScore = "0";
 
+				std::string& line_str = file->m_lineMap.at(n);
```

Comment:
Kcov now builds with c++17, so you can use auto & for these.

Reply:
got it.